
Support for position independent executable is detected but then GCC fails
to link the NTP daemon. Disable the PIE test for now.

diff -wpruN --no-dereference '--exclude=*.orig' a~/wscript a/wscript
--- a~/wscript	1970-01-01 00:00:00
+++ a/wscript	1970-01-01 00:00:00
@@ -299,7 +299,7 @@ def configure(ctx):
 
     cc_test_flags = [
         ('PIC', '-fPIC'),
-        ('PIE', '-pie -fPIE'),
+        #('PIE', '-pie -fPIE'),
         # this quiets most of macOS warnings on -fpie
         ('unused', '-Qunused-arguments'),
         # This is a useless warning on any architecture with a barrel
